### PR TITLE
Add start/stop/remove services

### PR DESCRIPTION
**Pull request by leehuwuj**

### CODE REVIEW

File: src/manager/app/controllers/setup.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import logging
+import os
+
+import requests
+
+ENVIRONMENT = os.getenv("ENVIRONMENT", "dev")
+
+DEFAULT_QDRANT_HOST = (
```
Comment (marcusschiesser) [inline src/manager/app/controllers/setup.py:8]: we shouldn't assume this is always qdrant - shouldn't the default by chromadb inside of the RAGapp container to keep it simple first?

File: src/manager/frontend/src/components/sections/createAgent.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+                            <FormControl>
+                                <Input {...field} />
+                            </FormControl>
+                            <FormDescription>
+                                Specify docker image for the agent, recommended to use the default image
+                            </FormDescription>
+                            <FormMessage />
+                        </FormItem>
+                    )}
+                />
```
Comment (marcusschiesser) [inline src/manager/frontend/src/components/sections/createAgent.tsx:55]: i think that's to technical for the UI - put it in the ENV variable?

File: src/manager/app/controllers/setup.py (hunk 1 above)
Comment (leehuwuj) [reply src/manager/app/controllers/setup.py:8]: we're using Qdrant right now, but starting with Chroma seems a lot simpler!

File: src/manager/frontend/src/components/sections/createAgent.tsx (hunk 2 above)
Comment (leehuwuj) [reply src/manager/frontend/src/components/sections/createAgent.tsx:55]: Yes, i think we can just use the default image. So generally, user just need specify app name to create a new app.

File: src/manager/frontend/src/components/sections/createAgent.tsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+                    control={form.control}
+                    name="name"
+                    render={({ field }) => (
+                        <FormItem>
+                            <FormLabel>Agent name</FormLabel>
+                            <FormControl>
+                                <Input {...field} />
+                            </FormControl>
+                            <FormDescription>
+                                Specify the name of the agent
```
Comment (marcusschiesser) [inline src/manager/frontend/src/components/sections/createAgent.tsx:34]: i guess this will be the host name?

Comment (leehuwuj) [reply src/manager/frontend/src/components/sections/createAgent.tsx:34]: No, I'm adding a prefix `ragapp-` (just for identification with other services)

File: entrypoint.sh [hunk 4]
```diff
@@ -17,12 +17,12 @@ if [[ -n "$BASE_URL" ]]; then
     sed -i '/<\/head>/i <script>window.ENV = {"BASE_URL":"'$BASE_URL'"};</script>' static/*.html static/**/*.html
 
     echo "Updated static files successfully!"
+    echo "Running application..."
+    exec fastapi run main.py --root-path $BASE_URL
```
Comment (marcusschiesser) [inline entrypoint.sh:21]: this one doesn't solve your CMD issue?
```suggestion
    exec "$@" --root-path $BASE_URL
```

**Review by marcusschiesser — APPROVED** [review]

**Pull request merged**